config/initializers/sentry-raven.rb

```diff
@@ -0,0 +1,7 @@
+require 'raven'
+
+Raven.configure do |config|
+  config.dsn = ENV.fetch('INTEGRATION_SENTRY_DNS')
```

Comment:
huh, that's a good question, I'm pretty sure I copy pasted the `config.dsn` part and it looks wrong. Double checking now.